gramps/plugins/webreport/narrativeweb.py

```diff
@@ -1585,7 +1585,7 @@ def build_url_fname(self, fname, subdir=None, uplink=False,
         The extension is added to the filename as well.
 
         Notice that we do NOT use os.path.join() because we're creating a URL.
-        Imagine we run gramps on Windows (heaven forbits), we don't want to
```

Comment:
The English this would be "heaven forbid!".  :)

Reply:
Yes, but that idea isn't chiefly British, nor English. What sort of ancestral outfit would this be were it to forward such absurdities, the mere thought of which beckons the mind. It is the very backslash in the domain that is European history and culture.